test/integration/targets/ansible-doc/fakemodule.output

```diff
@@ -8,7 +8,6 @@ OPTIONS (= is mandatory):
 
 - _notreal
         really not a real option
-        [Default: (null)]
```

Comment:
default should still be shown, even if the formatting changed

Reply:
Fixed in 8334afd14c07affaa518c86f8ad560375e4cfeb6.